addons/analytic/models/analytic_plan.py

```diff
@@ -145,17 +150,21 @@ def _compute_analytic_account_count(self):
         for plan in self:
             plan.account_count = len(plan.account_ids)
 
+    def _compute_all_children_ids(self):
+        for plan in self:
+            plan.all_children_ids = plan.children_ids + plan.children_ids.all_children_ids
```

Comment:
In order to avoid adding a new **recursive** field, couldn't we just execute this?
```sql
  SELECT parent.id, 
         array_agg(child.id) 
    FROM account_analytic_plan parent
    JOIN account_analytic_plan child ON child.parent_path LIKE parent.parent_path || '%' 
   WHERE parent.id IN ...
GROUP BY parent.id
```